app/views/users/settings.html.erb

```diff
@@ -60,10 +62,12 @@
   <br />
   <br />
   <% end %>
-
+  
+  <br />
+  <h4><b>I want to receive an email </b></h4>
 
   <div style="display: inline-flex; justify-content: space-between; width: 90%;">
-    <span>Do you want to receive a customized digest weekly?</span>
+    <span>For a weekly digest </span>
```

Comment:
I think the issue says `In a weekly digest`...not sure if it is a strict change...I think it could remain as it is now...

This is the same for line 84

Reply:
Yes, we'd prefer the exact wording:

```
I want to receive an email

- Whenever there are new posts on topics I follow
- In a daily digest
- In a weekly digest

```

Can you make these changes? Thank you!!!!

Reply:
Sure I'll make these changes. Thanks!

Reply:
@jywarren @RuthNjeri there was nothing corresponding to "Whenever there are new posts on topics I follow" earlier. I can take it up as a separate issue and add it in another PR if that's fine!

Reply:
Hi @Manasa2850 I think that it is meant to be added as part of the notifications under `I want to receive an email` here  https://github.com/publiclab/plots2/pull/9240/files#diff-60dea8a0195e299c872e9b749c642d0606caf5119e88bdfaab11fe378c46534dR67

I think there would be an entirely new notification switch though it's a great idea to follow that up in another issue to find out is there is a `digest:topics` for topics

Reply:
Aha, ummm I think what is happening here is that these three options are actually a toggle. We can have only one of the three states, because a digest is /instead of/ updates for every post. Does that make sense? Is that distinction something we can spin out into a new issue or add to the original? Right now, i think we can skip the first option, but eventually it'll be like "do i want digests OR emails for each post?"

Does that make sense?  Thanks and sorry for the confusion!

Reply:
Or perhaps we can explicitly say that in help text above these toggles? Something like:

`Email notifications are sent for each post in topics you follow. If you prefer to receive a digest, choose one of these options:`

Reply:
Sure @jywarren. For now I'll just add a help text. I'll create another issue for creating a toggle for the options and then work on that.
Thanks!

Reply:
@RuthNjeri there is no `digest:topics` for topics as of now. I've created an issue #9260 for that and will try solving it.